packages/lib/src/vis/shared/AxisSystem.module.css

```diff
@@ -13,19 +13,15 @@
 .axisSystem > svg {
   display: block;
   overflow: visible;
-
-  /* Avoid potential overflow caused by default intrinsic size of `svg` elements. */
-  /* Chrome/Edge require values greater than 0. */
-  width: 1px;
-  height: 1px;
 }
 
 .axis[data-type='abscissa'] {
   grid-area: bottom-axis;
 }
 
 .axis[data-type='ordinate'] {
-  grid-area: axis-grid;
+  grid-area: left-axis;
+  transform: translateX(100%);
```

Comment:
Moving left axis to correct CSS grid area and adding `transform` to compensate for the positioning of the `group` inside the SVG. Feels more logical than to position the axis in the middle area, after all.